timemachine/fe/free_energy.py

```diff
@@ -75,14 +75,19 @@ class PairBarPlots:
 
 
 @dataclass
-class SimulationResult:
+class PairBarResult:
     initial_states: List[InitialState]
-    result: PairBarResult
+    bar_results: List[BarResult]
```

Comment:
Nit: shape documentation: would it be valid / helpful to
* annotate the lengths `len(initial_states) == K`, `len(bar_results) == K - 1` in line comments
* possibly assert `len(initial_states) == len(bar_results) + 1` in a `__post_init__`?

Reply:
Added in https://github.com/proteneer/timemachine/pull/974/commits/a767963466b0e11820af8a60649dab0acaef8e78